doc/quickstart.rst

```diff
@@ -41,48 +42,44 @@ And run the f_min-function:
+use the whole of *SMACs* capability, a scenario_ object should be used.
+
+Commandline
+~~~~~~~~~~~
+There are a wrapper `cmdline_wrapper.py`, a scenario-file `scenario.txt` and a
+PCS-file `param_config_space.pcs` in the branin-folder.
 To run the example scenario, change into the root-directory of *SMAC* and type the following commands:
 
 .. code-block:: bash
 
     cd examples/branin
-    python ../../scripts/smac --scenario branin_scenario.txt
+    python ../../scripts/smac --scenario scenario.txt
 
-The python command runs *SMAC* with the specified scenario. The scenario file contains the following five lines:
+The python command runs *SMAC* with the specified scenario. The scenario file contains the following lines:
 
-    .. literalinclude:: ../examples/branin/branin_scenario.txt
+    .. literalinclude:: ../examples/branin/scenario.txt
 
 The **algo** parameter specifies how *SMAC* can call the function or evaluate an algorithm that *SMAC* is optimizing.
-An algorithm call by *SMAC* will look like this:
+This is further explained in the chapter about the `Target Algorithm Evaluator (TAE) <tae.html>`_.
+An algorithm call by *SMAC* is of the following format:
 
     .. code-block:: bash
 
         <algo> <instance> <instance specific> <cutoff time> <runlength> <seed> <algorithm parameters>
-        
-An exemplary call in our Branin example would be:
-
-    .. code-block:: bash
-
         python branin.py 0 0 999999999.0 0 1148756733 -x1 -1.1338595629 -x2 13.8770222718
-
-    The first two parameters after the ``branin.py`` do not matter for this example since no instances are needed for the function optimization.
-    For algorithm optimization, the first parameter holds the instance name on which the algorithm is evaluated and the second can provide extra information about the instance (rarely used).
-
-    The third parameter gives the cutoff time (maximal running time) an algorithm is allowed to run and the fourth the runlength (maximal number of steps).
-
-    The fifth parameter is the random seed which is followed by the algorithm/function parameters.
     
 The **paramfile** parameter tells *SMAC* which Parameter Configuration Space (PCS_)-file to use. This file contains a list of the algorithm's parameters, their domains and default values:
 
-    .. literalinclude:: ../examples/branin/branin_pcs.pcs
+    .. literalinclude:: ../examples/branin/param_config_space.pcs
 
     x1 and x2 are both continuous parameters. x1 can take any real value in the range [-5, 10], x2 in the range [0, 15] and both have the default value 0.
 
-The **run_obj** parameter specifies what *SMAC* is supposed to **minimize**. For this example we are optimizing the solution quality.
-For *SMAC* to be able to interpret the results of an algorithm run, a wrapper should return the results of the algorithm run as follows:
+The **run_obj** parameter specifies what *SMAC* is supposed to **minimize**. Here we optimize solution quality.
```

Comment:
SMAC minimizes, but we optimizes quality? I think we don't yet have a solution for this weird contradiction.

Reply:
Is this really a wrapper here?

Reply:
I'll go with optimize! What else would you call it, if not wrapper?